adding azure static cred rotation known issue (#2436)
hashicorp/web-unified-docs: content/vault/global/partials/important-changes/known-issue/azure-rotate-static-creds-delay.mdx

> ### Azure rotate static credentials cannot run in quick succession

Reviewer: ```suggestion
### Azure rotate static credentials cannot run in quick succession ((#azure-sync-issue))
```

Reviewer: We assign an explicit heading anchor so it's easier to link to specific entries from other pages

> <!-- BEGIN: Vault:=v2.x -->
> 
> | Change      | Status   | Vault edition | Affected version | Fixed version
> | ----------- | -------- | ------------- | ---------------- | -------------
> | Known issue | Closed   | Enterprise    | 2.0.0            | No
> 
> <!-- END: Vault:=v2.x -->

Reviewer: ```suggestion
<!-- BEGIN: Vault:=v2.x -->

| Change      | Status   | Vault edition | Affected version | Fixed version
| ----------- | -------- | ------------- | ---------------- | -------------
| Known issue | Closed   | Enterprise    | 2.0.0            | No

<!-- END: Vault:=v2.x -->

<!-- BEGIN: Vault:=v1.21.x -->

| Change      | Status   | Vault edition | Affected version | Fixed version
| ----------- | -------- | ------------- | ---------------- | -------------
| Known issue | Closed   | Enterprise    | 1.21.5            | No

<!-- END: Vault:=v1.21.x -->
```

Reviewer: If the problem appeared in 2.0, I'm assuming it appeared in 1.21.5?

Author: I would assume so as well

#### Issue

> Due to Azure's eventual consistency, running the static-rotate command in quick succession may orphan credentials.

Reviewer: ```suggestion
Running the `static-rotate` command in quick succession may orphan credentials because the consistency process on Azure servers cannot keep up:
```
Style correction: avoid possessives
Also, it's not entirely clear what "Due to Azure's eventual consistency" means. I'm guessing it's an issue of systems on the Azure side not being able to keep up?

Reviewer 2: What about something like:

Azure’s eventual consistency causes a brief delay in state propagation; if Vault attempts a rotation before Azure’s internal data is fully synchronized, Vault may fail to remove the old credential.

> A brief delay in state propagation occurs because Azure systems may not immediately synchronize internal data. When Vault attempts a rotation before synchronization completes, it may fail to remove the old credential. You must clean up orphaned credentials manually.

Author: I tried to take both of your inputs into account and landed on this. What do y'all think?

Reviewer: lgtm :)

Reviewer 2: I think we should explicitly say "eventually consistent" as that is the correct technical label for the Azure model as per [CAP theorem](https://www.geeksforgeeks.org/system-design/cap-theorem-in-system-design/)

For example:

```shell-session
$ vault write -f azure/rotate-role/my-role
```

#### Workaround

> Wait a few minutes before each static-rotate call.

Reviewer: ```suggestion
Wait a few minutes before each `static-rotate` call.
```
